Rework Next.js quick start guide (wizard)

inventarSarah opened the pull request:
<!-- Use this checklist to make sure your PR is ready for merge. You may delete any sections you don't need. -->

## DESCRIBE YOUR PR
Reworked the Next.js Getting Started Guide into a Quick Start Guide. 
See #12288 (and parent #11859)

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ ] Urgent deadline (GA date, etc.): <!-- ENTER DATE HERE -->
- [ ] Other deadline: <!-- ENTER DATE HERE -->
- [x] None: Not urgent, can wait up to 1 week+

## SLA

- Teamwork makes the dream work, so please add a reviewer to your PRs.
- Please give the docs team up to 1 week to review your PR unless you've added an urgent due date to it.
Thanks in advance for your help!

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## LEGAL BOILERPLATE

<!-- Sentry employees and contractors can delete or ignore this section. -->

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## EXTRA RESOURCES

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

chargome commented on platform-includes/getting-started-primer/javascript.nextjs.mdx:
```diff
@@ -1,24 +1,9 @@
-Features:
+<Expandable title="Are you developing with Turbopack?">
```
imo this should not be all up at the top but closer to the verify section

inventarSarah replied:
if the information isn't crucial to have from the start, I vote for moving it down too.
@lforst what do you think -- is this something the user should know right from the start or later?

inventarSarah replied:
I moved this Expandable under "Verify" for now -- what do you think?

chargome commented on platform-includes/getting-started-install/javascript.nextjs.mdx:
```diff
@@ -1,24 +1,25 @@
+You can always add or remove config options manually later if needed. Though, the earlier you set these up, the better.
 
-If the setup through the wizard doesn't work for you, you can also <PlatformLink to="/manual-setup/">set up the SDK manually</PlatformLink>.
+<Expandable permalink title="What does the installation wizard change inside your application?">
```
```suggestion
<Expandable title="What does the installation wizard change inside your application?">
```

lforst commented on docs/platforms/javascript/common/index.mdx:
```diff
@@ -16,7 +16,10 @@ Check out the other SDKs we support in the left-hand dropdown.
 ## Prerequisites
 
-* You need a Sentry [account](https://sentry.io/signup/) and [project](/product/projects/)
+You need:
+
+- A Sentry [account](https://sentry.io/signup/) and [project](/product/projects/)
+- Your application up and running
```
I don't fully see the value the 2nd bullet point adds 🤔 Is there a reason you added it?

inventarSarah replied:
I understand why you question this - it's probably obvious to most users that they need a working app. 
Why I added this:
- helps users understand the guide's scope: integrate within an existing app; we're not creating a new one 
- makes sure the user is ready to integrate the SDK and test right away; this makes sure their app is already working (when they run into issues, they may have an easier time troubleshooting)

So, it can be helpful, but it's not necessary. To be honest, I'm happy with either option
Let me know what you think :)

lforst commented on docs/platforms/javascript/common/index.mdx:
```diff
@@ -32,11 +35,33 @@ Select which Sentry features you'd like to install in addition to Error Monitori
 
+<PlatformSection supported={["javascript.nextjs"]}>
+
+## Step 1: Choose Your Sentry Features (Optional)
```
I would honestly remove this entire section. It's irrelevant to the setup in the docs since it's pretty much always gonna be the same because the wizard will already let you choose.

inventarSarah replied:
Please see my response to a similar comment here: https://github.com/getsentry/sentry-docs/pull/12291#discussion_r1914800148

lforst replied:
There are very conflicting opinions on adding a paragraph like this. IIRC we flipped back and fourth 4 times on having this paragraph vs not having it in the last 2 years.

I personally, don't think SDK onboarding is the place to go into that stuff. I do however see the value behind capturing people who know nothing about Sentry yet. How about instead of having a 3 point paragraph, we do a simple sentence like, "This guide will tell you how to hook up your application with the Sentry SDK. If you want to know more about Sentry itself go here."

inventarSarah replied:
Yes, it's a hot topic, as I've learned :)! There are already discussions about moving these functions to another location (e.g. an SDK landing page). And then, as you wrote, for example, link to it from the quick start page. 
However, since we currently don't yet have a place for it, I'd like to leave the information here.
Or do you know of a page we could link to? 
If not, is it okay for you to leave it like this for the time being?

I'm also working on the Next.js quick start guide for manual setup -> this page will also need to communicate the features to the user (including onboarding option checkboxes). How we will solve this in detail will probably also influence the quick start guide for the wizard setup (and vice versa).

lforst replied:
Makes sense, we can keep it for now!

lforst commented on platform-includes/getting-started-install/javascript.nextjs.mdx:
````diff
@@ -1,24 +1,25 @@
 ```
 
-The wizard will prompt you to log in to Sentry. It will then automatically do the following steps for you:
+The wizard starts guiding you through the setup process, which also includes enabling the Sentry features listed above.
````
We would need to remove the part about features if we remove the features section above.

lforst commented on platform-includes/getting-started-install/javascript.nextjs.mdx:
```diff
@@ -1,24 +1,25 @@
+<Expandable permalink title="What does the installation wizard change inside your application?">
 
-</Note>
+- Create config files with the default `Sentry.init()` calls for each runtime (node, browser, edge)
```
```suggestion
- Create config files with the default `Sentry.init()` calls for all runtimes (Node.js, Browser, and Edge)
```

inventarSarah replied:
updated 👍

lforst commented on platform-includes/getting-started-next-steps/javascript.nextjs.mdx:
```diff
@@ -0,0 +1,18 @@
+Now's a good time to customize your setup and look into more advanced topics.
+Our next recommended steps for you are:
+
+- Learn more about our [Vercel integration](/organization/integrations/deployment/vercel/)
+- Continue to [customize your configuration](/platforms/javascript/guides/nextjs/configuration/)
+- Learn how to [manually capture errors](/platforms/javascript/guides/nextjs/usage/)
+- Get familiar with [Sentry's product features](/product) like tracing, insights, and alerts
```
```suggestion
- Learn how to [manually capture errors](/platforms/javascript/guides/nextjs/usage/)
- Continue to [customize your configuration](/platforms/javascript/guides/nextjs/configuration/)
- Get familiar with [Sentry's product features](/product) like tracing, insights, and alerts
- Learn more about our [Vercel integration](/organization/integrations/deployment/vercel/)
```

shuffle based on priority

inventarSarah replied:
looks good, thanks!
updated the order of the list items

lforst commented on docs/platforms/javascript/common/index.mdx:
```diff
@@ -32,11 +35,33 @@ Select which Sentry features you'd like to install in addition to Error Monitori
+During setup, you'll have the option to enable specific Sentry features for your application.
+Here's a quick overview of what's what:
+
+- [**Error Monitoring**](/product/issues) (always enabled): Automatically report errors, uncaught exceptions, and unhandled rejections. If you have something that looks like an exception, Sentry can capture it.
+- **Tunneling**: Route browser requests to Sentry through a Next.js rewrite to bypass ad-blockers that might interfere with Sentry's data collection.
+- [**Tracing**](/product/tracing): Track software performance while seeing the impact of errors across multiple systems. For example, distributed tracing allows you to follow a request from the frontend to the backend and back.
+- [**Session Replay**](/product/explore/session-replay/web/getting-started/): Get to the root cause of an issue faster by viewing a video-like reproduction of what was happening in the user's browser before, during, and after the problem.
```
If we end up keeping this the order should be:

- Errors
- Tracing
- Replay
- Tunneling (thinking about it tunneling is a weird thing to have here and should probably go)

chargome replied:
+1 on removing tunneling

inventarSarah replied:
👍 removed Tunneling from the list

lforst commented on platform-includes/getting-started-primer/javascript.nextjs.mdx:
```diff
@@ -1,24 +1,9 @@
-- Events [enriched](/platforms/javascript/enriching-events/context/) with device data
-- [Breadcrumbs](/platforms/javascript/enriching-events/breadcrumbs/) created for outgoing HTTP request with XHR and Fetch, and console logs
-- [Release health](/product/releases/health/) for tracking crash-free users and sessions
+The Sentry SDK doesn't fully support `next dev --turbo` as Turbopack is still under development.
```
```suggestion
The Sentry SDK doesn't fully support `next dev --turbo` as Turbopack is still under development. This means that the Sentry SDK will not capture any data for the frontend. Other than that, your devserver should still be fully operational.
```

inventarSarah replied:
updated 👍 thank you!

lforst commented on platform-includes/getting-started-primer/javascript.nextjs.mdx:
```diff
@@ -1,24 +1,9 @@
+The Sentry SDK doesn't fully support `next dev --turbo` as Turbopack is still under development.
 
-<Note>
+However, Sentry will work fine in your production builds.
```
Since we're talking about dev here I think we could remove this line.

inventarSarah replied:
Yes, it makes sense to me -- I removed this line.

lforst commented on platform-includes/getting-started-verify/javascript.nextjs.mdx:
```diff
@@ -1,46 +1,30 @@
-Add a button to a frontend component that throws an error:
+## Step 3: Verify Your Setup
```
I don't like this section because the wizard will tell you to do this. We should avoid putting information in multiple places. If the wizard is missing information that we would have added here, we should add it to the wizard instead. My working theory is that once people run the wizard command, they will not look back into the docs until they are done with looking at the error in Sentry. At which point all of this text is already irrelevant. What's more important in that moment is the Next Steps.

inventarSarah replied:
I understand you're suggesting we simplify the guide to only include "Prerequisites," "Install," and "Next Steps." -- Did I get this right?

This is a new idea, and I'd like to share it with the others to get their input as well (hopefully tomorrow).

I see the advantages of having a shorter guide and relying more on the wizard.  
However, there are a few points we need to be mindful about:
- once the terminal window in which the wizard ran is closed, all its information is gone; the quick start guide is always accessible
- "they will not look back into the docs until they are done with looking at the error in Sentry" - on the other hand, we probably also have users who don't read all the text in the terminal (I'm such a candidate, for example). Especially everything that comes after the green Success message may be perceived as "not important"; ideally, we can make both of these user types happy
- many users who use this type of guide are probably new and don't know much about Sentry yet. So the quick start guide is a great place to introduce them to features beyond error monitoring and provide links to more information -> but we have also talked about providing this info somewhere else (eg. on a landing page #12289)


this is also relevant to your comment on the features: [/#discussion_r1910162943](https://github.com/getsentry/sentry-docs/pull/12291#discussion_r1910162943)

lforst replied:
These points make sense. We can keep it, but I would advocate for half a sentence like "If you haven't done so through the setup wizard, go through the following steps to....". That way people will know not to do the same thing twice.

inventarSarah replied:
Thank you -- I'll rewrite this section for sure because I now see that it can be misleading!

lforst commented on platform-includes/getting-started-verify/javascript.nextjs.mdx:
```diff
@@ -1,46 +1,30 @@
+Let's confirm that Sentry is working properly and sending data to your Sentry project by using the example page and route created by the installation wizard.
 
-And throw an error in an API route:
+1. Open the example page at [http://localhost:3000/sentry-example-page](http://localhost:3000/sentry-example-page)
```
another problem: it may not always be `localhost:3000` depending on the app

lforst commented on platform-includes/getting-started-verify/javascript.nextjs.mdx:
```diff
@@ -1,46 +1,30 @@
+<Alert level="warning" title="Important">
 
-Refer to <PlatformLink to="/manual-setup/#errors-from-nested-react-server-components">Capturing Errors From Nested React Server Components</PlatformLink> for detailed setup instructions.
+Errors triggered from within your browser's developer tools are sandboxed, so they will not trigger Sentry's error monitoring.
```
```suggestion
Errors triggered from within your browser's developer tools (i.e. the browser console) are sandboxed, so they will not trigger Sentry's error monitoring.
```

inventarSarah replied:
updated 👍

lforst reviewed (commented):
I love that we give this page some attention! Since this page is so critical though, we should be rather selective how much we bombard users with information. Additionally, the information we give them needs to be bulletproof. Anything in the wrong place will spark immense confusion.

chargome commented on docs/platforms/javascript/common/index.mdx:
```diff
@@ -32,11 +35,33 @@ Select which Sentry features you'd like to install in addition to Error Monitori
 
 </PlatformSection>
 
+<PlatformSection supported={["javascript.nextjs"]}>
```
I think excluding/including parts with `PlatformSection` will become (or actually already is) messy as we adapt other guides to this new structure. I guess the way forward will be to have this (step 1, step 2, ...) on every guide right? Anyway that's something we need to refactor afterwards

inventarSarah replied:
Yes, exactly -- over time, all quick start guides should use Step 1, Step 2, ...
Until then, let me know if there's anything I can already do to keep things from getting overly messy :)

chargome reviewed (commented):
Nice rework! 

Let's make sure to merge in https://github.com/getsentry/sentry-docs/pull/12293 before this one ~~and also see if @lforst has any comments here~~

inventarSarah commented:
@lfrost @chargome
For testing purposes, I have updated this quick start guide with some ideas from the team.

This also addresses some of your feedback, so I hope you can find some improvements in these changes too! 

What's changed:
- Step 1 should be Install (the old Step 1 wasn't actionable, which does not make sense for something called "Step" :D)
- Test: moved the features list into an Expandable in the Install section
- Test: moved the list in View Captured Data in Sentry into an Expandable
- Test: moved the Expandable about Turbopack into the Verify section


I like that the guide is more actionable now and that we can still include information without scaring the user with an overly long page (by using the Expandables). However, I  feel like there are too many boxes now and that it might not be pleasant to look at.
I need to revisit the page with refreshed eyes later again :)
In the meantime, please have a look if you want, and I will also ask Salma to do the same

Thank you!!

mydea commented on platform-includes/getting-started-install/javascript.nextjs.mdx:
```diff
@@ -1,24 +1,40 @@
-- create or update your Next.js config with the default Sentry configuration
-- create `.sentryclirc` with an auth token to upload source maps (this file is automatically added to `.gitignore`)
-- add an example page to your app to verify your Sentry setup
+<Expandable title="Want to learn a bit more about these features?">
```
```suggestion
<Expandable title="Want to learn more about these features?">
```

I would tend to leave the `a bit` away, but no strong feelings!

inventarSarah replied:
Let's remove it 👍

mydea commented on platform-includes/getting-started-install/javascript.nextjs.mdx:
```diff
@@ -1,24 +1,40 @@
-</Note>
+</Alert>
+
+You can always add or remove features manually later if needed. Though, the earlier you set these up, the better.
```
```suggestion
You can always add or remove features manually later if needed, but the setup is easiest if features are enabled from the start.
```

Not 100% sure, but it seems a bit "unclear" why it is better (from a users point of view). Maybe this is a bit clearer?

inventarSarah replied:
We could also remove the Alert ("This guide assumes that you enable all features ...") and instead write in basic text:

This guide assumes that you enable all features and allow the wizard to create an example page and route. You can add or remove features at any time, but setting them up now will save you the effort of configuring them manually later.

@mydea what do you think?

mydea replied:
sounds good to me!

inventarSarah replied:
Updated!

whitep4nth3r commented:
Hello! This is sooo much better!

There's one thing I noticed that could be clearer.

The page title for the wizard setup is 'Next.js' and the page title for the manual setup is 'Manual setup'. For (probably SEO purposes) and also for clarity, could we have 'Next.js' in the title for manual setup as well? So maybe the quick start title could be 'Next.js: Quick Start' and the manual setup title could be 'Next.js: Manual Setup'. When I was on the manual setup page, I had to look to the sidebar or the breadcrumbs to make sure I was in the right place.

inventarSarah commented:
> Hello! This is sooo much better!

Yes, 💯

> The page title for the wizard setup is 'Next.js' and the page title for the manual setup is 'Manual setup'. For (probably SEO purposes) and also for clarity, could we have 'Next.js' in the title for manual setup as well? So maybe the quick start title could be 'Next.js: Quick Start' and the manual setup title could be 'Next.js: Manual Setup'. When I was on the manual setup page, I had to look to the sidebar or the breadcrumbs to make sure I was in the right place.

Yes, we will definitely change the titles! For now, we need to keep them as they are because the title influences the name on the menu. For example, now the root page for next.js is our quick start guide called "Next.js". When we rename that, the new name will show up in the sidebar navigation but also in the overview lists, like on this page https://docs.sentry.io/

codyde commented:
I love how this is coming together. Really happy with it.

A suggestion to consider that i'd love @lforst perspective on - when I landed on this page; I wasnt entirely sure what was supported in Next.JS specifically (which is a super common question these days with things like RSC + Server Actions, as an example). We do a great job about calling out those configs in the Manual setup. 

I think theres a lot of value in a sentence on top that says somethign like

"Sentry has out of the box support for Pages and App router features. Features include React Server Components, Server Actions, Error Boundaries, Sourcemaps, and Tunneling" and link to the relevant content in each one of those. Just a suggestion; we can adjust the words. 

Much of that is handled in the wizard; but my thinking is if someone lands here - they'd get a clear answer. Feels like a positie the Next.JS experience.

lforst commented:
@codyde I think the reason why we never put such a section there is because we always supported everything (except for in transitionary periods). I never saw the value behind reciting what Next.js features there are. I question the value of explicitly stating what we support, when we support _everything_. I might be wrong though. I think the question that needs more answering is "What data should I see in Sentry (when everything is set up correctly)? And how do I use that data?"

inventarSarah commented:
> I think the question that needs more answering is "What data should I see in Sentry (when everything is set up correctly)? And how do I use that data?"

@lforst We could have a link to documentation/blog post/video ? that talks more about this in the Next Steps -- I'll see what I can find :)
Let me know if you have any more thoughts on this!

lforst commented:
> @lforst We could have a link to documentation/blog post/video ? that talks more about this in the Next Steps -- I'll see what I can find :)
> Let me know if you have any more thoughts on this!

A link to somewhere sounds good. Video not so much because you cannot update it incrementally. Other than that I think we can/should merge this PR soon. We can figure stuff out step by step.

whitep4nth3r reviewed (approved):
Great work. I love how the full guide fits completely on my weird vertical type monitor as a bonus. ✅

coolguyzone commented on platform-includes/getting-started-install/javascript.nextjs.mdx:
```diff
@@ -1,24 +1,34 @@
 
-After the wizard setup is completed, the SDK will automatically capture unhandled errors, and monitor performance.
-You can also <PlatformLink to="/usage/">manually capture errors</PlatformLink>.
+- [**Error Monitoring**](/product/issues) (always enabled): Automatically report errors,
```
I know Issues is our error monitoring product, but clicking on 'Error Monitoring' and being directed to the 'Issues' page might be confusing for those less familiar with Sentry. 

What do you think about having this anchor text say 'issues' and providing some context in the description?

e.g. "Issues (always enabled): Sentry's core error monitoring product that automatically reports errors . . ."

inventarSarah replied:
Let's do as you suggested! This is also a good idea because we also write about this "Issues" page under "Need help locating the captured errors in your Sentry project?"

Updated it 👍

coolguyzone commented on platform-includes/getting-started-install/javascript.nextjs.mdx:
```diff
@@ -1,24 +1,34 @@
+<Expandable title="What does the installation wizard change inside your application?">
 
-</Alert>
+- Create config files with the default `Sentry.init()` calls for all runtimes (Node.js, Browser, and Edge)
```
```suggestion
- Creation of config files with the default `Sentry.init()` calls for all runtimes (Node.js, Browser, and Edge)
```

inventarSarah replied:
I kept the verbs active (with an -s ending to match the expandable title) instead of using -ion. This sounds simpler and less formal to me -- what do you think? 

(also, same for similar suggestions below)

coolguyzone replied:
Ok, good call!

coolguyzone commented on platform-includes/getting-started-install/javascript.nextjs.mdx:
```diff
@@ -1,24 +1,34 @@
 
-</Alert>
+- Create config files with the default `Sentry.init()` calls for all runtimes (Node.js, Browser, and Edge)
+- Add a Next.js instrumentation hook to your project (`instrumentation.ts`)
```
```suggestion
- Addition of Next.js instrumentation hook to your project (`instrumentation.ts`)
```

coolguyzone commented on platform-includes/getting-started-install/javascript.nextjs.mdx:
```diff
@@ -1,24 +1,34 @@
-</Alert>
+- Create config files with the default `Sentry.init()` calls for all runtimes (Node.js, Browser, and Edge)
+- Add a Next.js instrumentation hook to your project (`instrumentation.ts`)
+- Create or update your Next.js config with the default Sentry settings
```
```suggestion
- Creation of or updates to your Next.js config with the default Sentry settings
```

coolguyzone commented on platform-includes/getting-started-install/javascript.nextjs.mdx:
```diff
@@ -1,24 +1,34 @@
+- Create config files with the default `Sentry.init()` calls for all runtimes (Node.js, Browser, and Edge)
+- Add a Next.js instrumentation hook to your project (`instrumentation.ts`)
+- Create or update your Next.js config with the default Sentry settings
+- Create `.sentryclirc` with an auth token to upload source maps (this file is automatically added to `.gitignore`)
```
```suggestion
- Creation of a `.sentryclirc` with an auth token to upload source maps (this file is automatically added to `.gitignore`)
```

coolguyzone commented on platform-includes/getting-started-install/javascript.nextjs.mdx:
```diff
@@ -1,24 +1,34 @@
+- Add a Next.js instrumentation hook to your project (`instrumentation.ts`)
+- Create or update your Next.js config with the default Sentry settings
+- Create `.sentryclirc` with an auth token to upload source maps (this file is automatically added to `.gitignore`)
+- Add an example page and route to your application to help verify your Sentry setup
```
```suggestion
- Addition of an example page and route to your application to help verify your Sentry setup
```

coolguyzone commented on platform-includes/getting-started-verify/javascript.nextjs.mdx:
```diff
@@ -1,46 +1,42 @@
+<Expandable level="warning" title="Are you developing with Turbopack?">
 
-And throw an error in an API route:
+The Sentry SDK doesn't fully support `next dev --turbo` as Turbopack is still under development. This means that the Sentry SDK will not capture any data for the frontend. Other than that, your devserver should be fully operational.
```
```suggestion
The Sentry SDK doesn't fully support `next dev --turbo` as Turbopack is still under development. This means that the Sentry SDK will not capture any data from your frontend. Other than that, your dev server should be fully operational.
```

inventarSarah replied:
updated this

coolguyzone commented on platform-includes/getting-started-verify/javascript.nextjs.mdx:
````diff
@@ -1,46 +1,42 @@
-  res.status(200).json({ name: "John Doe" });
-};
-```
+If you haven't tested your setup yet, let's do it now. You can confirm that Sentry is working properly and sending data to your Sentry project by using the example page and route created by the installation wizard.
````
```suggestion
If you haven't tested your Sentry configuration yet, let's do it now. You can confirm that Sentry is working properly and sending data to your Sentry project by using the example page and route created by the installation wizard:
```

inventarSarah replied:
updated 👍

coolguyzone commented on platform-includes/getting-started-verify/javascript.nextjs.mdx:
```diff
@@ -1,46 +1,42 @@
+If you haven't tested your setup yet, let's do it now. You can confirm that Sentry is working properly and sending data to your Sentry project by using the example page and route created by the installation wizard.
 
-<Alert>
+1. Open the example page `/sentry-example-page` in your browser.
```
```suggestion
1. Open the example page `/sentry-example-page` in your browser (for most Next projects this will be on localhost).
```

inventarSarah replied:
👍 updated with minor adjustment

coolguyzone commented on platform-includes/getting-started-next-steps/javascript.nextjs.mdx:
```diff
@@ -0,0 +1,18 @@
+## Next Steps
+
+You should now have integrated Sentry into your Next.js application and should already be sending error and performance data to your Sentry project.
```
```suggestion
At this point you should now have integrated Sentry into your Next.js application and should already be sending error and performance data to your Sentry project.
```

coolguyzone reviewed (approved):
Hey @inventarSarah, this looks great! I added some comments/suggestions, feel free to push back/reject any of them. 🏄‍♂️

inventarSarah commented on platform-includes/getting-started-install/javascript.nextjs.mdx:
```diff
@@ -1,24 +1,34 @@
+- [**Tracing**](/product/tracing): Track software performance while seeing the
+  impact of errors across multiple systems. For example, distributed tracing
+  allows you to follow a request from the frontend to the backend and back.
+- [**Session Replay**](/product/explore/session-replay/web/getting-started/):
```
Yes, good idea! 
Updated 👍

codyde reviewed (approved):
This looks great! I looked over the other feedback and saw you incorporated it all. This is all good on my side.

LGTM!
